changelogs/fragments/49267-docker-facts-modules.yaml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - docker_container_facts - created docker_container_facts module
```

Comment:
I think I simply forgot the `container` in the changelog fragments filename; but adding `docker_network_facts` here too doesn't hurt ;-)

Reply:
I couldn't find any changelog fragment for `docker_container_facts` in the /changelog/fragments/ directory, which is why I've added it here. Putting all the new `_facts` modules in here together just makes sense to me, especially if they're all under the same Issue.

Reply:
I think I got a bit confused here. I just remembered that opposed to new features, new module PRs must not have a changelog fragment. All new modules are automatically listed in the changelog, so there's no need to manually add changelog fragments for them. Can you remove this file?